addons-l10n/ja/project-source-button.json

```diff
@@ -0,0 +1,5 @@
+{
+    "project-source-button/@name":"プロジェクトソース表示ボタン",
+    "project-source-button/@description":"プロジェクトのソースのJSONファイルやアセットを表示するボタンを追加する",
+    "project-source-button/hover":"プロジェクトソースビューワーを開く"
+}
```

Comment:
You shouldn't be editing this file directly. Please delete it

Reply:
I fixed it. Thank you.